package.json

```diff
@@ -30,6 +30,7 @@
     "eslint-config-airbnb": "18.2.1",
     "eslint-config-prettier": "8.0.0",
     "eslint-import-resolver-babel-module": "5.2.0",
+    "eslint-import-resolver-typescript": "^3.6.0",
```

Comment:
let's pin this (you'll need to run `yarn` again before merging the PR)
```suggestion
    "eslint-import-resolver-typescript": "3.6.0",
```

Reply:
Fixed this.